ScarletEngine/Thirdparty/tracy/client/TracyProfiler.hpp

```diff
@@ -565,7 +663,7 @@ class Profiler
     {
         assert( len <= TargetFrameSize );
         bool ret = true;
-        if( m_bufferOffset - m_bufferStart + len > TargetFrameSize )
+        if( m_bufferOffset - m_bufferStart + (int)len > TargetFrameSize )
```

Comment:
shouldn't we use static_cast ?

Reply:
Yes :)

Reply:
This is thirdparty code though and we'll likely be updating our version of their code frequently so I don't really want to modify it every time